README.md

```diff
@@ -1,153 +1,169 @@
+  - [Bootstrap](#bootstrap-1)
+    - Node
+    - Cluster
+  - [Lifecycle](#lifecycle-1)
+    - Node
+    - Cluster
+
+The rest of the the document details the **cluster management** features that were
+briefly listed above. The design choices to establish these features are captured
+[here.](management/DESIGN.md)
 
 ##Inventory
-Inventory management requires:
-- **Cluster membership management**: It should be possible to automatically discover the nodes
-and track their discovery status.
-- **Node state database**: It should be possible to track the current state of the node in
-the cluster. A few possible states are node has been discovered but waiting to be
-commissioned; node has been commissioned; node is being upgraded; and node has
-been decommissioned.
+Inventory management provides:
+- **Cluster Membership Management**: Automatically discovers nodes and tracks discovery status.
+- **Node State Database**: Tracks the current state of the node within the cluster.
+Node states consist of the following:
+
+  - **New**: A node that has not completed the bootstrap and discovery process.
+  - **Discovered**: A node has completed bootstrapping and notifies the Cluster Manager. The node waits to be Commissioned.
+  - **Commissioned**: A node has been configured to participate in the cluster.
+  - **Upgraded**: One or more node components have been successfully upgraded and the node is ready to participate in the cluster.
+  - **Decommissioned**: All configuration and software components have been removed and the node no longer participates in the cluster.
 
 ##Node Image Management
-Node image management requires:
-- **Image repository**: It should be possible to have the base image available at a central
-repository. The image can then be made available to be booted using a system like pxe.
-- **Image installation**: It should be possible to automate image installation on new nodes
-with minimal user involvement as provided by kickstart.
-- **Image upgrade**: It should be possible to trigger cluster wide or rolling image upgrades
-from a central location.
+Node Image Management provides:
+- **Image Repository**: A central location reachable by nodes, where images are hosted. Images are delivered to nodes using mechanisms such as PXE.
+- **Image Installation**: Automates the image installation of new nodes.
+- **Upgrades**: Automates the process of upgrading the node image. Upgrades can be automatically triggered and can be performed cluster-wide or rolling.
 
-**Note:** To start simple this feature is **not** provided as part of initial
-release of cluster manager. In the initial release the operator shall be provided the base image
-as a bootable iso file that they will need to integrate with their node onborading process.
+**Note:** Node Management features are **not** included in the initial release of cluster manager.
+In the initial release the operator will be provided the base image as a bootable iso file
```

Comment:
Done. Here is the wording:

**Note:** Node Image Management features are **not** included in the initial release of Contiv Cluster.
Operators are responsible for node bootstrapping, including image provisioning. The
[`node discovery` workflow]
(https://github.com/contiv/cluster/blob/master/management/baremetal.md#3-provision-rest-of-the-nodes-for-discovery-from-the-control-host)
should be followed to provision nodes with infra services and clustering software to complete the cluster bootstrap.